components/pages/explorer/QueryActions/ShareButton.tsx

```diff
@@ -0,0 +1,72 @@
+import { useTheme } from 'emotion-theming';
+import { css } from '@emotion/core';
+import {
+  FacebookIcon,
+  FacebookShareButton,
+  LinkedinIcon,
+  LinkedinShareButton,
+  TwitterIcon,
+  TwitterShareButton,
+} from 'react-share';
+
+import Dropdown from '@/components/Dropdown';
+import { DefaultTheme } from '@/components/theme';
+import { ShareIcon } from '@/components/theme/icons';
+import { Fragment } from 'react';
+
+const ShareButton = ({
+  queryLabel,
+  queryUrl,
+  title,
+}: {
+  queryLabel: string;
+  queryUrl: string;
+  title?: string;
+}) => {
+  const theme: DefaultTheme = useTheme();
+
+  const shareChannels = [
+    <FacebookShareButton quote={`COVID sequence data: ${queryLabel}`} url={queryUrl}>
+      <FacebookIcon size="13" /> Share on Facebook
+    </FacebookShareButton>,
+
+    <TwitterShareButton title={`COVID sequence data: ${queryLabel}`} url={queryUrl}>
+      <TwitterIcon size="13" /> Share on Twitter
+    </TwitterShareButton>,
+
+    <LinkedinShareButton title={`COVID sequence data: ${queryLabel}`} url={queryUrl}>
+      <LinkedinIcon size="13" /> Share on Linkedin
+    </LinkedinShareButton>,
+  ];
+
+  return (
+    <Dropdown
+      css={css`
+        .react-share__ShareButton {
```

Comment:
had to customise these buttons because they didn't like being nested in such small space.